src/main/java/temporal/SimpleWorkflowImpl.java

```diff
@@ -0,0 +1,77 @@
+package temporal;
+
+import io.temporal.activity.ActivityOptions;
+import io.temporal.api.enums.v1.ParentClosePolicy;
+import io.temporal.workflow.Async;
+import io.temporal.workflow.ChildWorkflowOptions;
+import io.temporal.workflow.Workflow;
+import java.time.Duration;
+import java.util.LinkedList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import temporal.model.OrderInfo;
+
+public class SimpleWorkflowImpl implements SimpleWorkflow {
+
+  private boolean exit = false;
+
+  private static final Logger logger = LoggerFactory.getLogger(SimpleWorkflowImpl.class);
```

Comment:
Replace `LoggerFactory.getLogger` with `Workflow.getLogger` - the Workflow logger should be used within Workflow classes

Reply:
@reselbob - please address this

Reply:
addressed